dotcom-rendering/src/web/server/populatePageTemplate/index.ts

```diff
@@ -0,0 +1,62 @@
+import type { EmotionCache } from '@emotion/cache';
+import createEmotionServer from '@emotion/server/create-instance';
+import {
+	BUILD_VARIANT,
+	dcrJavascriptBundle,
+} from '../../../../scripts/webpack/bundles';
+import type { DCRNewslettersPageType } from '../../../types/newslettersPage';
+import { pageTemplate } from '../pageTemplate';
+import { buildWindowGuardian } from './makeWindowGuardian';
+import { getScriptTags } from './scriptTags';
+
+export const populatePageTemplate = (
+	model: DCRNewslettersPageType,
```

Comment:
I can see that this PR looks to simplify the page template function to make it more approachable which I think is great, but given that it's still limited to the DCRNewsletterPageType, perhaps we should update the naming scheme of the folder to reflect that it's only for newsletters?

Again I think there's room to improve the existing DCR solution here!